frontend/src/components/layout/navigation/MobileNavigation.module.scss

```
@@ -3,16 +3,62 @@
 
 .mobile-menu {
   width: 100%;
-  position: sticky;
-  top: 0;
   // `height: 0` prevents the menu from taking up space when it's not expanded:
   height: 0;
-  // z-index of 1 (so it overlaps content)
-  // allows menu to slide under main navigation header
-  z-index: 1;
   flex-direction: column;
 }
 
+.nav-mask-phone {
+  width: 100%;
+  position: absolute;
+  display: flex;
+  flex-direction: row;
+  border-bottom: 1px solid $color-light-gray-20;
+  background-color: white;
+
+  > * {
+    flex: 1 1 auto;
+    padding: $spacing-lg 0 $spacing-md 0;
+    justify-content: center;
+    display: flex;
+    border-bottom: 5px solid transparent;
+
+    &.is-active {
+      border-bottom: 5px solid $color-purple-40;
+    }
```

How about this to add a hover state?

```suggestion
    }

    &:hover {
      background-color: $color-purple-05;
    }
```

---

Eduardo said we didn't need hover states for mobile, but i'll add that in just incase.

---

Yeah, the thing is that we're not detecting whether someone is on mobile, but whether they have a small screen. So it might be the case that someone is on a small screen, has resized the window, or that they're on mobile but have issues with touch input and are using another input device. And I think maybe screen readers might show hover states as well, for people with low-but-not-no vision? If we'd have only displayed this menu with `@media not (any-pointer: fine)`, then maybe we could be somewhat confident that they wouldn't be necessary, but generally I'd default to having both hover and focus styles for all interactive elements just to be sure.